### ISSUE TITLE
whether the target domain dataset should be divided into three parts: training, validation and test?

### ISSUE THREAD

**Issue by sunshehai** [CityU-AIM-Group/SIGMA#35]
Hello, I noticed that your target domain dataset is only set into training set and test set, but after starting the validation after 100 iterations, whether the target domain dataset should be divided into three parts: training, validation and test, so as to adapt the unsupervised domain to target detection.

**Comment by wymanCV** [comment]
Hi, thanks for your interest. The dataset splittings all follow previous benchmarks for far comparison.  Kindly refer to the [survey](https://ieeexplore.ieee.org/abstract/document/10075484?casa_token=vikalzBJuqoAAAAA:kyCK3NA-4c0RMFNcLU6V0k23XlctxMV61dLv6jbBYCsnAo-SuyZXx197szYQJ-JCqUbCD68) for more information. Hope this will help.
